src/tools/cephfs/cephfs-shell

```diff
@@ -1108,6 +1108,78 @@ sub-directories, files')
                 self.poutput('{:10s} {}'.format(humansize(int(cephfs.getxattr(to_bytes(
                     dir_), 'ceph.dir.rbytes').decode('utf-8'))), '.' + dir_))
 
+    quota_parser = argparse.ArgumentParser(
+        description='Quota management for a Directory')
+    quota_parser.add_argument('op', choices=['get', 'set'],
+                              help='Quota operation type.')
+    quota_parser.add_argument('dir', type=str, help='Name of the directory.')
+    quota_parser.add_argument('--max_bytes', type=long, default=-1, nargs='?',
```

Comment:
Get this error in QA testing:

````
2019-04-17T21:21:47.416 INFO:tasks.cephfs.test_cephfs_shell:Running command: cephfs-shell -c /etc/ceph/ceph.conf -- help
2019-04-17T21:21:47.416 INFO:teuthology.orchestra.run.smithi039:Running:
2019-04-17T21:21:47.416 INFO:teuthology.orchestra.run.smithi039:> cephfs-shell -c /etc/ceph/ceph.conf -- help
2019-04-17T21:21:47.743 INFO:teuthology.orchestra.run.smithi039.stderr:Traceback (most recent call last):
2019-04-17T21:21:47.744 INFO:teuthology.orchestra.run.smithi039.stderr:  File "/usr/bin/cephfs-shell", line 304, in <module>
2019-04-17T21:21:47.744 INFO:teuthology.orchestra.run.smithi039.stderr:    class CephFSShell(Cmd):
2019-04-17T21:21:47.744 INFO:teuthology.orchestra.run.smithi039.stderr:  File "/usr/bin/cephfs-shell", line 938, in CephFSShell
2019-04-17T21:21:47.744 INFO:teuthology.orchestra.run.smithi039.stderr:    quota_parser.add_argument('--max_bytes', type=long, default=-1, nargs='?',
2019-04-17T21:21:47.744 INFO:teuthology.orchestra.run.smithi039.stderr:NameError: name 'long' is not defined
````

`long` no longer exists in python3. See also: https://stackoverflow.com/a/2104947